helpers/update-site/documentation.js

```diff
@@ -169,14 +171,14 @@ const getResourcesFiles = async () => {
     const imageFiles = [];
 
     for (const resource of resources) {
-        const images = await globby([`${constants.dirs.NODE_MODULES}/{vscode-webhint,@hint/${resource}-*,@hint/configuration-all/node_modules/{vscode-webhint,@hint/${resource}-*}}/images/**/*`], { absolute: true });
+        const images = await globby([`${documentationGlobbyPattern}/images/**/*`], { absolute: true });
```

Comment:
This will now match the `hint` package whereas it didn't before. Is that intended or do we need to exclude `hint` either in the pattern or the results to avoid picking up extra resources?

Reply:
I think we shouldn't match `hint` here. The hint package is processed in another place IIRC.

Reply:
I've tested the process locally and all the documentation is looking good.
Personally I think we can keep it as is.

Reply:
Works for me if @sarvaje is okay with it.

Reply:
Yep, if everything looks good, I'm ok :)